components/contributors/GithubActivity.tsx

```diff
@@ -387,6 +396,11 @@ export default function GithubActivity({ activityData }: Props) {
   };
 
   const [activityTypes, setActivityTypes] = useState([...ACTIVITY_TYPES]);
+  const [storedFilters, setStoredFilters] = useAtom(activityTypesAtom);
+
+  useEffect(() => {
+    if (storedFilters) setActivityTypes(storedFilters);
+  }, [storedFilters]);
```

Comment:
This useEffect won't be needed when atom itself is used to manage the state in the first place

```suggestion
```

Reply:
Okay, updating it asap!